src/Aquarius.ONE.ClientSDK/Enterprise/Core/CoreApi.cs

```diff
@@ -460,10 +460,15 @@ public async Task<List<ProductOffering>> GetProductOfferingsAsync(EnumProductOff
                     expand = "?expand=tenant,feature";
                     break;
             }
+
+            var tenantIdQueryParameter = string.IsNullOrEmpty(tenantId)

```

Comment:
I felt like it would be easier to read if I made a new variable for the tenant id - one part of the query string for the 'expand' parameter, one part for the 'tenantId' parameter, etc.